checkov/gitlab/checks/two_factor_authentication.py

```diff
@@ -24,9 +24,9 @@ def __init__(self) -> None:
     def scan_entity_conf(self, conf: list[dict[str, Any]], entity_type: str) -> tuple[CheckResult, list[dict[str, Any]]] | None:  # type:ignore[override]
         if schema.validate(conf):
             for group in conf:
-                if group.get("require_two_factor_authentication", False) is True:
-                    return CheckResult.PASSED, conf
-            return CheckResult.FAILED, conf
+                if group.get("require_two_factor_authentication") is False:
```

Comment:
is `require_two_factor_authentication` always set and can't be `None`? or is the default `True`?

Reply:
Yes it is set. I think it defaults to False, since creating a group without it yields `require_two_factor_authentication=False`